Core/PixelEvent.swift

```diff
@@ -482,6 +483,7 @@ extension Pixel.Event {
             
         case .jsAlertShown: return "m_js_alert_shown"
         case .jsAlertBlocked: return "m_js_alert_blocked"
+        case .jsAlertBackgroundTap: return "m_js_alert_bg_tap"
```

Comment:
jsAlertBlocked is not being called, maybe it should be renamed to jsAlertTabClosed = "m_js_alert_tab_closed" since we don't have the block option anymore?

jsAlertBackgroundTap also not being called, when is this supposed to be called?